.github/test-a-feature.md

```diff
@@ -103,4 +103,69 @@ yarn test
 yarn test:watch
 ```
 
-[1]: https://github.com/stardust-ui/react/tree/master/test/specs/commonTests
\ No newline at end of file
+[1]: https://github.com/stardust-ui/react/tree/master/test/specs/commonTests
+
+## Behaviours tets
+
+Behavior unit tests are generated from the specification written in each behavior file.
+Each line under the "@specification" tag is taken and trying to match the regex experesion written in "testDefinitions.ts" file. 
+
+**Adding test(s)**
+
```

Comment:
1. For a new behavior:
- ...
- ...
2. For an existing behaviour:
- ...
- ...